advanced_alchemy/service/_util.py

```diff
@@ -147,11 +155,16 @@ def to_schema(
 
         Args:
             data: The return from one of the service calls.
+              Type: :class:`~advanced_alchemy.repository.typing.ModelOrRowMappingT`
             total: The total number of rows in the data.
-            filters: Collection of route filters.
-            schema_type: Optional schema type to convert the data to
+            filters: :class:`~advanced_alchemy.filters.StatementFilter` or :class:`sqlalchemy.sql.expression.ColumnElement` Collection of route filters.
```

Comment:
My previous comment on types being optional applies here as well. Whatever approach we decide, I feel it should be consistent. Currently some have types, some don't.